src/App.tsx

```diff
@@ -15,47 +15,100 @@ export const goodsFromServer = [
   'Garlic',
 ];
 
+enum SortType {
+  NONE = 0,
+  ALPHABET = 1,
+  LENGTH = 2,
+}
+
+function getReorderedGoods(
+  goods: string[],
+  isReversed: boolean,
+  sortType: SortType,
+):string[] {
+  const visible = [...goods];
```

Comment:
use better naming for the variables 
```suggestion
  const visibleGoods = [...goods];
```